### DISCUSSION TITLE
Missing libdfm package

### DISCUSSION THREAD

**Discussion by wanmukhlis** [linuxdeepin/developer-center#5059] [Q&A | 问答板块]
i tried to run make dde-file-manager and got dependencies issue for libdfm-io, libdfm-mount and libdfm-burn. i managed to install util-dfm package but still encountered the same issue. please advice
![2](https://github.com/linuxdeepin/developer-center/assets/29225381/240e07aa-df68-4ab2-8dae-73e52075cece)

**Comment by BLumia** [comment]
If you are trying to build the `master` branch source from an older system (v20 stable), then it will require dependencies that the system repo doesn't have. 

Suggestions:

1. If you insist to build the newest version, then try install v23 beta or using NixOS/Nix
2. If you just want to hack into the source code that not need to be the latest version, then try `apt source dde-file-manager` and use that source, or check the version and `git checkout` to that tag release.
